### DISCUSSION TITLE
Noob question : Unable to configure the integration

### DISCUSSION THREAD

**Discussion by tokyoseyes** [CJNE/ha-myenergi#458] [Q&A]
Hi there !

I have a problem in the plugin's configuration: MyEnergi credentials won't work in the UI, i tried with 2 different accounts, same thing happen : Wrong username or password (i double check, they works fine on myenergi website).
I tried to uninstall/reinstall the plugin without any success.

Any ideas ?

![Capture d’écran du 2023-12-21 20-37-09](https://github.com/CJNE/ha-myenergi/assets/57300596/480b486d-5237-4dfc-9b66-d655bc754267)

**Comment by G6EJD** [comment]
You need hub serial number and the API key you get from the web portal.
